spec/features/views_recipe_maker_page_spec.rb

```diff
@@ -32,14 +24,14 @@
     in my batch, so that I can make estimations about the OG and IBU.
     }do
       visit new_recipe_path
-      fill_in 'Recipe Title' , with: 'The Best Pale Ale'
-      fill_in 'Batch Size in Gallons:', with:'5'
-      click_on 'Next Step'
+      fill_in 'Name', with: 'The Best Pale Ale'
+      fill_in 'Batch size', with: '5'
+      click_on 'Add Recipe'
 
-      expect(page).to have_content('5 Gal. Batch')
+      expect(page).to have_content('The Best Pale Ale')
```

Comment:
If you assign a FactoryGirl object to a variable and call the name here, it would be easier. Short of that, just instantiate a new recipe and call on the name. Avoid using the same string in multiple places. Creates the possibility for bugs that are hard to catch in your tests.